ExternalSampleProjects/publish.sh

```diff
@@ -0,0 +1,17 @@
+for Sample in ExternalSampleProjects/*/
+do
+	echo Running ${Method} on ${Sample}
+	mkdir -p ${Sample}Assets/ExternalSamplesUtility/Editor
+	ln -s ../../Packages ${Sample}Packages
```

Comment:
I assume this won't work correctly. Our manifest from the project doesn't mention the input system as it is an "embedded" (or whatever it is called) package in `Packages/`.

Reply:
That's why we don't copy the manifest file, but link the whole directory.

Reply:
Ah right, so we can load it without depending on the package. Didn't realize that. Makes me wonder, how do we get manifest.json including the input system reference into the .unitypackage then?

Reply:
We can't get the input system reference into the unitypackage. But: If you get the unitypackage into a project using the samples mechanism, then you already have the Input System package.

Reply:
The user could install that .unitypackage anywhere, though, including a fresh project. But guess we're kinda out of options there. Not surprised .unitypackage and package manager don't mix. Hope that eventually `Samples` matures into a mechanism that can handle all this for us.

Reply:
Yes, indeed. OTOH, the discovery mechanism for these packages makes it more likely for the user to put them into a project which has the Input System package.